Mathlib/Analysis/NormedSpace/Exponential.lean

```diff
@@ -473,6 +473,12 @@ theorem exp_continuous : Continuous (exp 𝕂 : 𝔸 → 𝔸) := by
   exact continuousOn_exp
 #align exp_continuous NormedSpace.exp_continuous
 
+open Topology in
+lemma _root_.Filter.Tendsto.NormedSpace_exp {α : Type*} {l : Filter α} {f : α → 𝔸} {a : 𝔸}
```

Comment:
I don't think `NormedSpace_exp` is compatible with the naming convention. `exp_normedSpace` looks better, although not great.

Reply:
The naming convention is not really explicit on how to deal with names containing a namespace like `NormedSpace.exp` when they show up as part of lemma names. Unfortunately, `Filter.Tendsto.exp` is already claimed by the real exponential function.
Would `NormedSpace.Filter.Tendsto.exp` be a reasonable alternative?

Reply:
You really want to be able to use dot notation, so it has to be `Filter.Tendsto.something`. The something could be `normedSpaceExp` or `exp_normedSpace` or `exp_of_normedSpace`. Maybe the best option is indeed `normedSpaceExp` because it's a single atom, and we need a way to separate it from the other ones. Or just `exp`, and use `rexp` and `cexp` for the real and complex version (I think that's my favorite option).

Reply:
I'm renaming `Filter.Tendsto.exp` to `Filter.Tendsto.rexp` and then `Filter.Tendsto.NormedSpace_exp` to `Filter.Tendsto.exp`.